src/magic_folder/__main__.py

```diff
@@ -8,9 +8,5 @@
 )
 
 if __name__ == '__main__':
-    from pkg_resources import load_entry_point
-    import sys
-
-    sys.exit(
-        load_entry_point('magic-folder', 'console_scripts', 'magic-folder')()
-    )
+    from magic_folder.cli import _entry
+    _entry()
```

Comment:
I guess this is slightly unrelated, but it is to allow gridsync to use this file as entrypoint directly, rather than to provide its own (https://github.com/gridsync/gridsync/commit/1d6a33f7a05c553833d4f4b7adafc316e6b055fb).